middlewares/auth_middleware.go

```diff
@@ -18,6 +18,16 @@ type Claims struct {
 	jwt.RegisteredClaims
 }
 
+// if Status = development you can bypass all routes
+var Status string = func() string {
+	_ = godotenv.Load("../.env")
+	s := os.Getenv("STATUS")
+	if s == "" {
+		s = "production"
+	}
+	return s
+}()
```

Comment:
Prefer lazily evaluating this global value; for example, with `sync.OnceValue`:

```suggestion
var Status = sync.OnceValue(func() string {
	_ = godotenv.Load("../.env")
	s := os.Getenv("STATUS")
	if s == "" {
		s = "production"
	}
	return s
})
```